Actions/Copy.cs

```diff
@@ -1,14 +1,16 @@
-﻿#if UNITY_EDITOR
+﻿
 namespace UnityEditor.Experimental.EditorVR.Actions
 {
     [ActionMenuItem("Copy", ActionMenuItemAttribute.DefaultActionSectionName, 5)]
     sealed class Copy : BaseAction
     {
         public override void ExecuteAction()
         {
+#if UNITY_EDITOR
```

Comment:
The cut/copy/paste/duplicate commands we can probably simulate - instantiate a deactivated object on copy (in the clip board), instantiate activate duplicates on paste.

Reply:
Yeah, we used to do it that way. Worked fine